meta/runtime.yml

```diff
@@ -1,2 +1,2 @@
 ---
-requires_ansible: ">=2.9.10"
+requires_ansible: "<=2.17.0"
```

Comment:
Version smaler that the current one? are you expecting this to break with a nex Ansible release?

Reply:
No but that is the highest stable version which is covered in our tests. Also the 2.9.10 also works but linter says it should be used a supported version in this variable. You mean we should use >=2.15.0?

Reply:
I don't link this change. That would imply that we need to update this with every new Ansible version. I would only like to update things here if stuff actually breaks.

Maybe disable this rule like we did in dev-sec: https://github.com/dev-sec/ansible-collection-hardening/blob/master/.config/ansible-lint.yml#L20

Reply:
Yes, i think that would be the best. Thank you, I changed it